lib/User.py

```diff
@@ -23,14 +23,12 @@ class UserNotFoundError(Exception):
 class User(UserMixin):
     def __init__(self, id, auth_instance):
         """Simple User class"""
-        if not Configuration.loginRequired():
-            # dummy account for when logon is not required.
-            self.id = "_dummy_"
```

Comment:
Have you tested the possible impact this has? Is the admin section still accessible with loginRequired set to false? IOW is the original authentication workflow still supported?

Reply:
@P-T-I, Yes I did and it works. In fact since the dummy is passed to the func from the view, I have just refactored to be able to use the login_user for the OIDC flow same as the dummy to avoid changing the routes while disabling the default auth handler in this case since auth was handled by the IDP.

Reply:
Cool!